bin/problem.py

```diff
@@ -155,11 +147,8 @@ def maybe_copy(x):
             return x.copy() if copy and isinstance(x, (list, dict)) else x
 
         samplesonly = only_sample
-        try:
-            if config.args.samples:
-                samplesonly = True
-        except AttributeError:
-            pass
+        if config.args.samples:
+            samplesonly = True
```

Comment:
merge with the line above: `samplesonly = config.args.samples or only_sample` (ugh these variable names aren't good)

Reply:
Hehe, I'll just leave the variable names as-is for now, I don't really have a really good suggestion anyway. :joy: But I merged the two lines :slightly_smiling_face: (57be5a8)